test/jdk/java/awt/print/PrinterJob/PrintGlyphVectorTest.java

```diff
@@ -97,54 +120,48 @@ public int print(Graphics g, PageFormat pf, int pageIndex) {
         return Printable.PAGE_EXISTS;
     }
 
+    private static void createTestUI() {
+        Frame f = new Frame("Test PrintGlyphVector");
+        PrintGlyphVectorTest pvt = new PrintGlyphVectorTest();
+        f.add(pvt, BorderLayout.CENTER);
+
+        Button printButton = new Button("PRINT");
+        PrinterJob pj = PrinterJob.getPrinterJob();
+        if (pj == null || pj.getPrintService() == null) {
+            System.out.println("Printer not configured or available."
```

Comment:
I realise the test was already doing the pj == null check but per. the spec, this shouldn't happen
https://docs.oracle.com/en/java/javase/17/docs/api/java.desktop/java/awt/print/PrinterJob.html#getPrinterJob()

Reply:
Right, and IDE generates a warning: _`pj == null` is always false_. I suggest removing this condition and thus resolving the warning.

Reply:
Just a bit of optimisation: you can postpone creating the test frame. The frame is not needed if the test passes because there's no print service.